Dockerfile

```diff
@@ -12,3 +12,10 @@ ENTRYPOINT ["./entrypoint.sh"]
 
 FROM base as dev
 ENTRYPOINT ["poetry", "run", "flask", "run", "-h", "0.0.0.0", "-p", "5000"]
+
+FROM base as test-reload
+ENTRYPOINT ["poetry", "run", "ptw", "--poll"]

```

Comment:
Neat, this even seems to work over a bind mount on Windows, most file watchers I've tried haven't